pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "rgbmatrix"
+version = "0.0.1+20250323"
```

Comment:
I think we would want to avoid hard-coding this, since it would require an update to the package version each time a new commit is merged. Forgetting to update this value is really bad, since it means pip might not pick up changes.

We should look at [hatch-vcs](https://github.com/ofek/hatch-vcs) to version based on git SHA instead. I haven't used this personally, but sounds like it does exactly what we need:

```
  [build-system]
  requires = ["hatchling", "hatch-vcs"]

  [tool.hatch.version]
  source = "vcs"
```

This should let us dynamically define the version, something like `0.0.1+g8f3a1c2`. We'll need to check it -- I believe it uses latest tag + SHA by default.